Automattic/wp-calypso
# Remove: Boosted Post survey

We got the data we needed, turning it off.

To test:
1. Go to one of your sites and view your posts.
1. Add `?tour=boostedPostSurvey` to the end of the URL.
1. Boosted post dialog should _not_ appear.
